### ISSUE TITLE
Using Lerna?

### ISSUE THREAD

**Issue by EtherTyper** [voxel/voxel-engine-stackgl#11]
I think it would be more convenient to store all of these npm packages in a [Lerna monorepo](https://github.com/lerna/lerna). That way, downloading the source for development or forking the project would be much easier. It would also help if they all shared a common versioning scheme.

**Comment by deathcap** [comment]
@EtherTyper Agreed, also related: https://github.com/voxel/voxel-engine-stackgl/issues/9. Lerna looks promising, or some kind of monorepo. I did write and use this tool to help manage committing updates to module dependencies: https://github.com/deathcap/lmno-cl but it still was cumbersome to update all the different repos. Not to mention the spam caused by mass updates across many repos (I may have noticed some interested users unsubscribe from notifications because of this). In retrospect, this was one of the larger problems with this project, or at least my use of it (especially with https://github.com/voxel/voxelmetaverse, which almost entirely consists of references to other npm packages).
